modules/unsupported/javafx/README.md

```diff
@@ -5,6 +5,8 @@ A JavaFX parent subclass containing a Canvas. A map from a WMS-Server is drawn t
 
 In development.
 
+Note: You need to [download OpenJDK](https://wiki.openjdk.java.net/display/OpenJFX/Download) to use this module.
```

Comment:
Why so? Should work with Oracle too given the dependencies in the pom. Afaik it was not working with OpenJDK, not even before (OpenJDK did not package JavaFX)